test/structure/structure_io/test_pdb_io.py

```diff
@@ -31,7 +31,7 @@
 )
 
 # pylint: disable=protected-access, invalid-name
-_LOGGER.setLevel(logging.DEBUG)
+_LOGGER.setLevel(logging.CRITICAL)
```

Comment:
This is something I add for debug. Just commenting it out will be fine as CRITICAL is the default level.

Reply:
Sure. It just made it difficult to read through testing output

Reply:
Changed. Now the line `_LOGGER.setLevel(logging.DEBUG)` is just commented out